src/icons.yml

```diff
@@ -5256,12 +5256,15 @@ icons:
     categories:
       - Web Application Icons
 
-  - name:       Viacoin
+  - name:       Viacoin (VIA)
     id:         viacoin
     unicode:    f237
     created:    4.3
     url:        viacoin.org
+    aliases:
```

Comment:
This is not necessary

Reply:
Bitcoin is listed as **Bitcoin (BTC)**, other currencies are also listed with their short symbols such as **Euro (EUR)**

Reply:
I wasn't talking about the whole update, just that the alias is not needed as it's the same as the id

Reply:
i see :)
so it should be probably changed as follows?
id to **via** and alias to **viacoin**